### PR TITLE
[Bug Fix] Incorrect line width caused by whitespaces

### LINKED ISSUE #40: [Bug Report] Wrong line width when used with fortune
`fsays` failed to format long string when used with piped fortune.

```text
└─(09:51:39 on wenqing ✹)──> fortune | fsays                                                                                                                                                          
 _________________________________________
/       My dear People.                         \
|       My dear Bagginses and Boffins, and my   |
| dear Tooks and Brandybucks,             |
| and Grubbs, and Chubbs, and Burrowses,  |
| and Hornblowers, and Bolgers,           |
| Bracegirdles, Goodbodies, Brockhouses   |
| and Proudfoots.  Also my good           |
| Sackville Bagginses that I welcome back |
| at last to Bag End.  Today is my        |
| one hundred and eleventh birthday: I am |
| eleventy-one today!"                    |
\               -- J. R. R. Tolkien                     /
 -----------------------------------------
        \
         \
            _~^~^~_
        \) /  o o  \ (/
          '_   -   _'
          / '-----' \
```

### PR DESCRIPTION
**Pull request by WenqingZong**
Aims to fix https://github.com/rust-lang/ferris-says/issues/40

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -88,8 +90,11 @@ where
     // Final output is stored here
     let mut write_buffer = SmallVec::<[u8; BUFSIZE]>::new();
 
+    // Pre process to merge continues white spaces into one space character while preserving newline characters.
```
Comment (lcnr) [inline src/lib.rs]:
```suggestion
    // Pre process to merge continuous whitespaces into one space character
    // while preserving newline characters.
```

**Review by lcnr — COMMENTED** [review]
LGTM! A small nit and then this is ready for merge!

File: src/lib.rs (hunk 1 above)
Comment (WenqingZong) [reply src/lib.rs]: Fixed!

**Comment by lcnr** [comment]
Thank you!

**Pull request merged**